snowshu/adapters/source_adapters/snowflake_adapter.py

```diff
@@ -295,11 +296,10 @@ def quoted(val: str) -> str:
     @overrides
     def _build_conn_string(self, overrides: Optional[dict] = None) -> str:  # noqa pylint: disable=redefined-outer-name
         """overrides the base conn string."""
-        conn_parts = [f"snowflake://{self.credentials.user}:{self.credentials.password}"
-                      f"@{self.credentials.account}/{self.credentials.database}/"]
```

Comment:
Is it worth quoting almost all of them to avoid any similar future issues?

Reply:
Potentially yes, was thinking about that as well